src/full/Agda/TypeChecking/Constraints.hs

```diff
@@ -135,23 +135,31 @@ whenConstraints action handler =
     stealConstraints pid
     handler
 
+wakeConstraints' :: (ProblemConstraint -> TCM Bool) -> TCM ()
```

Comment:
I don't want to nitpick but *some* comment for the extra argument here would be nice.